rest_sensors.cpp

```diff
@@ -2510,13 +2510,23 @@ void DeRestPluginPrivate::checkSensorStateTimerFired()
                 }
                 else if (!item && sensor->modelId() == QLatin1String("lumi.vibration.aq1") && sensor->type() == QLatin1String("ZHAVibration"))
                 {
+                    //Set tiltangle to 0 by defaut but without QueueEvent
```

Comment:
Not good: you need to send an event on every change, or rules won't trigger and clients that don't poll won't see the change.